sklearn/feature_selection/_sequential.py

```diff
@@ -227,12 +227,12 @@ def fit(self, X, y=None, **params):
             Returns the instance itself.
         """
         _raise_for_params(params, self, "fit")
-        tags = self._get_tags()
+        tags = self.__sklearn_tags__()
         X = self._validate_data(
             X,
             accept_sparse="csc",
             ensure_min_features=2,
-            ensure_all_finite=not tags.get("allow_nan", True),
+            ensure_all_finite=not tags.input_tags.allow_nan,
```

Comment:
Similarly here, it seems like we want to default to `True`

Reply:
previously we assumed the tag could simply not exist, but now the tag always exists.